app/decorators/service_provider_session_decorator.rb

```diff
@@ -11,12 +11,14 @@ def initialize(sp:, view_context:, sp_session:, service_provider_request:)
     @service_provider_request = service_provider_request
   end
 
-  delegate :redirect_uris, to: :sp, prefix: true
```

Comment:
just to check my understanding, this isn't saving a query, this is just making a method definition more explicit, right?

Reply:
Yeah, I got a bit frustrated trying to find where `sp_redirect_uris` was defined since it's a few layers beneath `decorated_session`